README.md

```diff
@@ -56,43 +61,21 @@ Received 2 packets, got 1 answers, remaining 0 packets
 '192.30.253.113'
 ```
 
-### Python module
-
-It is straightforward to use Scapy as a regular Python module, for example to
-check if a TCP port is opened.
-First, save the following code in a file names e.g. `send_tcp_syn.py`
-
-```python
-from scapy.all import *
-conf.verb = 0
-p = IP(dst="github.com")/TCP()
-r = sr1(p)
-print(r.summary())
-```
-
-Then, launch the script with:
-
-```python
-sudo python send_tcp_syn.py
-IP / TCP 192.30.253.113:http > 192.168.46.10:ftp_data SA / Padding
-```
-
 ### Resources
 
-To begin with Scapy, you should check [the notebook
-hands-on](https://github.com/secdev/scapy/blob/master/doc/notebooks/Scapy%20in%2015%20minutes.ipynb)
-and the [interactive
-tutorial](http://scapy.readthedocs.io/en/latest/usage.html#interactive-tutorial).
-If you want to learn more, see [the quick demo: an interactive
-session](http://scapy.readthedocs.io/en/latest/introduction.html#quick-demo)
-(some examples may be outdated), or play with the
-[HTTP/2](doc/notebooks/HTTP_2_Tuto.ipynb) and [TLS](doc/notebooks/tls)
-notebooks.
-
-The [documentation](http://scapy.readthedocs.io/en/latest/) contains more
+The [documentation](https://scapy.readthedocs.io/en/latest/) contains more
 advanced use cases, and examples.
 
-## Installation
+Other useful resources:
+
+- [Scapy in 20 minutes](https://github.com/secdev/scapy/blob/master/doc/notebooks/Scapy%20in%2015%20minutes.ipynb)
```

Comment:
Codacy complains about missing newlines after each list element.

Reply:
Should be fixed by now, thanks for the call